dispatch/static/manager/src/js/components/ContentEditor/embeds/VideoEmbed.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+
+import { FormInput, TextInput } from '../../inputs'
+
+class VideoEmbedComponent extends React.Component {
+
+  constructor(props){
+    super(props)
+
+    this.handleUrlChange = this.handleUrlChange.bind(this)
+    this.insertVideo = this.insertVideo.bind(this)
+
+    this.state = {
+      inserted: false,
+      validURL: false
+    }
+  }
+
+  handleUrlChange(e){
+    e.preventDefault()
+    this.props.updateField('url', e.target.value)
+    console.log("URL IS: "+this.props.data.url);
+
+  }
+
+
+  insertVideo(){
+    var id = this.validYouTube(this.props.data.url);
+    this.props.updateField('id',id)
+    console.log("ID IS: "+this.props.data.id)
+
+    if(this.props.data.id){
+      this.setState({
+        validURL: true,
+        inserted : true
+      })
+    }
+  }
+
+  validYouTube(url){
+     var p = /^(?:https?:\/\/)?(?:www\.)?youtube\.com\/watch\?(?=.*v=((\w|-){11}))(?:\S+)?$/
```

Comment:
Is it deliberate that this does not accept YouTube short urls, which are fairly common? (eg. the "Share" panel gives you short urls, with the pattern `https://youtu.be/<ID>`).

example:
![screen shot 2017-02-26 at 10 34 41 pm](https://cloud.githubusercontent.com/assets/887504/23351115/07066f0c-fc74-11e6-8baa-d0e4574ca12f.png)